src/hex_tarball.erl

```diff
@@ -0,0 +1,455 @@
+-module(hex_tarball).
+-export([create/2, unpack/2, format_checksum/1, format_error/1, gzip/1, normalize_requirements/1]).
```

Comment:
gzip/1 and normalize_requirements/1 are only exposed for testing (gzip is currently tested in Hex, I'll move property test over in a separate PR). What's the best way to expose functions just for tests and not in prooduction?

Reply:
Wrap it in an `ifdef`. rebar3 sets `TEST` when running tests so you can do:

```erlang
-ifdef(TEST).
-export([gzip]).
-endif.
```